service-worker.js

```diff
@@ -0,0 +1,22 @@
+---
+---
+
+importScripts('/scripts/sw-toolbox.js');
+
+const VERSION = 1;
```

Comment:
How will you make sure that `VERSION` is incremented whenever any existing content anywhere on the site is updated? If `VERSION` doesn't get incremented, then the previously cached content for a given URL will never get refreshed, since there's a `cacheFirst` policy being used.